architecture/core_concepts/routes.adoc

```diff
@@ -1181,3 +1183,69 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+[[disable-namespace-ownership-check]]
+== Disabling the Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace `ns1` creates the oldest route r1 with host `*_www.abc.xyz_*`, then it
+owns that host name and subdomain (`abc.xyz`) for wildcard routes.  If another
+namespace `ns2` tries to create a route with say a different path `*_www.abc.xyz/path1/path2_*`,
+it would fail because a route in another namespace (`ns1` in this case) owns
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[wildcard routes]
+as the namespace that owns the subdomain owns all hosts in the subdomain.
+If a namespace owns subdomain `*.abc.xyz*`, anouther namespace can't claim `z.abc.xyz`.
+
+By disabling the namespace ownership rules you can disable these restrictions
+and allow hosts (and subdomains) to be claimed across namespaces.
+
+[WARNING]
+====
+If you decide to disable the namespace ownership checks in your router,
+be aware that this allows end users to claim ownership of hosts
+across namespaces. While this change can be desirable in certain
+development environments, use this feature with caution in production
+environments and ensure that your cluster policy has locked down untrusted end
+users from creating routes.
+====
+
+For example, with `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true`, if
+namespace `ns1` creates the oldest route `r1`  `*_www.abc.xyz_*`,  it owns only
+the hostname (+ path).  Another namespace can create a wildcard route
+even though it does not have the oldest route in that subdomain (`abc.xyz`)
+and we could potentially have other namespaces claiming other
+non-wildcard overlapping hosts (for example, `foo.abc.xyz`, `bar.abc.xyz`,
+`baz.abc.xyz`) and their claims would be granted.
+
+Any other namespace (for example, `ns2`) can now create
+a route `r2`  `*_www.abc.xyz/p1/p2_*`  and it would be admitted.  Similarly
+another namespace (`ns3`) can also create a route  `wildthing.abc.xyz`
+with a subdomain wildcard policy and it can own the wildcard.
+
+So the policy `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true` is more
+lax and allows claims across namespaces.  The only time the router would
+reject a route with the namespace ownership disabled is if the host+path
+is already claimed.
+
+For example: if a new route `rx` tries to claim `*_www.abc.xyz/p1/p2_*`  - then that
+would be rejected as route `r2` owns that host+path combination.  Also, the `rx`
+route could be in namespace `ns2` or a new or other namespace called `meds`.
+(The namespace is arbitrary.) The route is rejected as a duplicate/host[+path]
```

Comment:
"rejected as a duplicate/host[+path] was already claimed" is confusing.  Do you mean "rejected as a duplicate claim because the host+path is already claimed"?